Dockerfile-dc

```diff
@@ -0,0 +1,21 @@
+FROM alpine:3.10
+
+RUN apk add -U --no-cache ca-certificates
+
+FROM golang:latest
+
+ADD . /app
+WORKDIR /app
+RUN go get -u github.com/gobuffalo/packr/v2/packr2
```

Comment:
We are using a fixed version of packr2, `GO111MODULE=on go install github.com/gobuffalo/packr/v2/packr2` should be enough - see the makefile for reference

Reply:
Will do!